README.md

```diff
@@ -95,7 +95,7 @@ ie.
 We create the file postgresql-admin-pw (no extension).
 Within the file we have one line of characters which is the secret.
 
-####MAC OS - Post Catalina
```

Comment:
@paul-graham - this is adding an unnecessary space at line end.

Reply:
I thought this was deliberate - a markdown thing to force a line feed.

Reply:
Hmm... could be - though, the "Pre Catalina" heading above doesn't have this.

Reply:
Fair point. I've requested clarification from @ernestman28 as it was one of his commits that added the space. Might just be a keyboard mishap like my extra new line yesterday! 😂